src/ess/spectroscopy/types.py

```diff
@@ -147,3 +117,9 @@ class InstrumentAngles(
 
 NXspeFileName = NewType('NXspeFileName', str)
 NXspeFileNames = NewType('NXspeFileNames', list[NXspeFileName])
+
+SampleEvents = NewType('Sampleevents', sc.DataArray)
```

Comment:
```suggestion
SampleEvents = NewType('SampleEvents', sc.DataArray)
```
Not sure it matters?